src/components/molecules/timeline-events/order-edit/confirmed-difference-due.tsx

```diff
@@ -1,50 +1,70 @@
-import { useAdminOrderEdit } from "medusa-react"
-import React from "react"
+import { useAdminOrder, useAdminOrderEdit } from "medusa-react"
```

Comment:
general thought on this one: 

I think we should consider showing the refund timeline event regardless of any OrderEdits. The approach would be this:

There is always a check for `order.paid_total > order.total` if the check is >0 then we show the refund timeline event. 

The narrative here is: 
We don't care how this discrepancy between paid and actual total arose, but we owe the customer money so now we should issue a refund. 

I will move this thought to an issue that we can consider separately, justed wanted to note it here.

Reply:
Added a generic timeline event for refunds that is rendered always on top of the list if there is refundable amount in the last commit, let me know what you think :)